tcmur_cmd_handler.c

```diff
@@ -747,6 +865,9 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	case WRITE_VERIFY_16:
 		ret = handle_write_verify(dev, cmd);
 		break;
+	case WRITE_SAME_16:
```

Comment:
Might as well also add the WRITE_SAME 0x41 (10), WRITE_SAME_32 0x0d (32) since its all pretty much the same code.